cnd/src/main.rs

```diff
@@ -121,10 +122,14 @@ fn main() -> anyhow::Result<()> {
         macaroon_path: settings.lightning.lnd.readonly_macaroon_path.clone(),
     };
 
-    // Han protocol
+    // RCF003 protocol
+    let rfc003_alpha_ledger_states = Arc::new(rfc003::LedgerStates::default());
+    let rfc003_beta_ledger_states = Arc::new(rfc003::LedgerStates::default());
+    let swap_communication_states = Arc::new(SwapCommunicationStates::default());
+
+    // Han/HErc20 protocols (A.K.A split protocols)
```

Comment:
:wrench: :clown_face: 
```suggestion
    // HAN/HErc20 protocols (A.K.A split protocols)
```